content/integrations/kubernetes.md

```diff
@@ -5,9 +5,47 @@ kind: integration
 git_integration_title: kubernetes
 ---
 
+# Overview
+
 Get metrics from your Kubelets in real time to:
 
-* Visualize your Kubernetes cluster performance
```

Comment:
Ah it's wrong in dogweb, and i was syncing up text across them. Fixed in both now.